wee_slack.py

```diff
@@ -3324,13 +3324,15 @@ def unfurl_blocks(message_json):
             elif block["type"] == "actions":
                 block_text.append("|".join(i["text"]["text"] for i in block["elements"]))
             elif block["type"] == "call":
-                block_text.append(". Join via ").append(block["call"]["v1"]["join_url"])
+                block_text.append(". Join via " + block["call"]["v1"]["join_url"])
             elif block["type"] == "divider":
                 block_text.append("\n")
             elif block["type"] == "context":
                 block_text.append("|".join(i["text"] for i in block["elements"]))
+            elif block["type"] == "rich_text":
+                continue
             else:
-                block_text.append(json.dumps(block))
+                dbg("DEBUG: {}").format(json.dumps(block))
```

Comment:
The first `)` should be after `format` instead of after `dbg`. The way it is now, you are calling `format` on the return value of `dbg`, which is `None`. That is why you would get the raw json block without the `rich_text` condition (it would raise an exception and go to the exception handler below).

Though I don't think a debug message like this with just the json contents is useful. It should also include a text saying something about unknown block type. I also think we should render `<Unsupported block type "type">` in the message (I mentioned this in the other PR too, [here](https://github.com/wee-slack/wee-slack/pull/729#discussion_r380413554)).